specification/k3po.control/SPEC.md

```diff
@@ -192,7 +174,7 @@ N/A
 ##### NOTIFIED event
 
 ###### Description
-This event is sent by the driver when a barrier is triggered.
+This event notifys the client that a barrier has been triggered.  It may be sent after a barrier is triggered, or after a AWAIT command if the barrier was previously completed.
 
```

Comment:
NOTIFIED is sent also as a response to NOTIFY, no matter what is the status of the barrier prior to the NOTIFY command.